spec/models/change_password_form.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+RSpec.describe ChangePasswordForm, type: :model do
+  context 'passowrd form' do
+    it 'must have old_password' do
+      form = ChangePasswordForm.new({old_password: nil, password: 'password1', password_confirmation: 'password1'})
+      form.valid?
+      expect(form.errors.full_messages[0]).to eq('Old password can\'t be blank')
+    end
+
+    it 'must have password' do
+      form = ChangePasswordForm.new({old_password: 'password', password: nil, password_confirmation: 'password1'})
+      form.valid?
+      expect(form.errors.full_messages[0]).to eq('Password can\'t be blank')
+    end
+
+    it 'must have password confirmation' do
+      form = ChangePasswordForm.new({old_password: 'password', password: 'password1', password_confirmation: nil})
+      form.valid?
+      expect(form.errors.full_messages[0]).to eq('Password confirmation can\'t be blank')
+    end
+
+    it 'password and password confirmation must match' do
+      form = ChangePasswordForm.new({old_password: 'password', password: 'password1', password_confirmation: 'password2'})
+      form.valid?
+      expect(form.errors.full_messages[0]).to eq('Password confirmation doesn\'t match Password')
+    end
+
+    it 'password must have at least 8 characters' do
+      form = ChangePasswordForm.new({old_password: 'password', password: '1234567', password_confirmation: '1234567'})
+      form.valid?
```

Comment:
would be better to be doing this instead of `form.valid?`
```suggestion
      expect(form.valid?).to eq(false)
```